prebid-mobile/prebid-mobile-rendering.md

```diff
@@ -0,0 +1,110 @@
+---
+
+layout: page_v2
+title: Rendering Module
+description: Description of how Prebid SDK supports rendering
+sidebarType: 2
+
+---
+
+# Rendering Module Overview
+
+> ⚠️ NOTE: Rendering is an Open Beta Feature
+
+Prebid SDK introduces the rendering facilities as a standalone module. Adding this module won't change anything in your current integration. 
+
+![How Prebid Mobile Works]({{site.baseurl}}/assets/images/prebid-mobile/rendering/modularization-roadmap-open-beta.png){: .pb-lg-img :}
+
+In the scope of the Open Beta release, you can use the Core and Rendering modules alongside each other. The current integration will work as is without changes but if you want to add or switch to the rendering integration you will have to make changes in the codebase.  
+
+The rendering module consist of three libraries:
+
+- **PrebidSDKRendering**, which introduces the classes for integration into UI and rendering engine
+- [OPTIONAL] **PrebidSDKGAMEventHandlers**, which wraps GAM classes in order to support GAM integration. 
+- [OPTIONAL] **PrebidSDKMoPubAdapters**, which wraps MoPub Adapters for Prebid Rendering module
+
+Using rendering module you can integrate next ad formats:
+
+- **Dispaly** - Banner/Interstitial
+- **Outsrteam Video** - Banner/Interstitial/Rewarded
+- **Native** - Disaply / Video
+
+Intefration steps are convinient and described in the respective docs for [iOS](TBD) and [Android](TBD). 
```

Comment:
Integration steps are described in the respective environment docs for [iOS](TBD) and [Android](TBD).